src/useFocusTrap.js

```diff
@@ -63,14 +63,11 @@ export function useFocusTrap() {
         const shiftKeyPressed = !!evt.shiftKey;
         let focusableElems = Array.from(
           trapRefElem.querySelectorAll(focusableElementsSelector)
+        ).filter(
+          (focusableElement) => focusableElement.getAttribute("tabindex") >= 0
```

Comment:
I think this one is a potential troublemaker. What if no `tabindex` is defined e.g. `a[href]` ?

Reply:
f3dcf1fbe6b487c64ab44c819341d19fb1fdd162